gravitee-apim-gateway/gravitee-apim-gateway-flow/src/test/java/io/gravitee/gateway/jupiter/flow/BestMatchFlowBaseTest.java

```diff
@@ -263,6 +263,30 @@ public static Iterable<Object> data() {
                     "/book/:bookId/chapter/:chapterId",
                     "/book/9999/chapter/145",
                 },
+                {
+                    List.of("/", "/api_entrypoint/some_path/:test", "/api_entrypoint/some_path/sub_path"),
+                    Operator.STARTS_WITH,
+                    "/api_entrypoint/some_path/sub_path",
+                    "/api_entrypoint/some_path/sub_path",
```

Comment:
Should we have another test by trying with `/api_entrypoint/some_path/sub_path/sub_sub_path` showing that the "STARTS_WITH" is actually well handled ?

```
{
                    List.of("/", "/api_entrypoint/some_path/:test", "/api_entrypoint/some_path/sub_path"),
                    Operator.STARTS_WITH,
                    "/api_entrypoint/some_path/sub_path",
                    "/api_entrypoint/some_path/sub_path/sub_sub_path",
}
```

Reply:
Sure, will add it!